### ISSUE TITLE
Could the npc_kill enpoint be extended to include chest loot drops from ToB / CoX?

### ISSUE THREAD

**Issue by tylersfd2** [llamaXc/osrs-events#3]
I would like to use the existing endpoint to get drop information when clan members get an unique from the ToB or CoX chests. 
Currently, the endpoint only supports data from killed NPCs. 

It would be nice if the npc_kill endpoint would send this information to my API! 

Can this functionality be implemented to the existing endpoint?

**Comment by llamaXc** [comment]
Yes that is a possibility. I will look into by end of week and get an
answer back!

**Comment by eggins** [comment]
I'd like to also extend this to ToA given the timeframe on how long ago this request was opened. Leave it with me for a bit and I'll see what I can do.
